examples/vdb/main.tf

```diff
@@ -9,41 +9,20 @@ terraform {
 
 provider "delphix" {
   tls_insecure_skip = true
-  key               = "1.XXXX"
-  host              = "HOSTNAME"
+  key               = "1.089N1yoNUoHis8cJqA6BHRaf5OnS1HWmDAlQgxzNmhxNESamd9T4CNuLyvjw8eVF"
```

Comment:
This example file contains test values..

Reply:
thanks, reverting